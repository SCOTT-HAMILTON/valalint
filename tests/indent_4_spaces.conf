[Checks]
space_indent = 4

[Result]
errors=24
warnings=0
applied_fixes=24
failed_fixes=0
