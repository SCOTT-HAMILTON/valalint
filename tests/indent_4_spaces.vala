namespace Test {

    /* Toplevel namespace - shift -1 */
    public void foo() {
        warning("Hello");
    }

public void bar(int foo) {
    switch (foo) {
        /* case - shift -1 */
        case 1:
            print(1);
            break;
        /* default - shift -1 */
        default:
            print(0);
            break;
    }

    /* &&, || - shift -1 */
    if (foo == 1
        || foo == 2
        && foo == 3) {
        print();
    } else {
    print();
    }

    printf("foo %d %d",
    1,
    2);
    print(print(
    1, 2));
}

} // namespace Test
