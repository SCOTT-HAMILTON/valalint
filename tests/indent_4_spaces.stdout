Vala lint failed: 21 error(s), 0 warning(s)
Lint auto fix succeeded: 21 fix(es) applied.
Some errors were fixed by auto fix. Run valalint again to get up-to-date results.
